static/app/views/alerts/rules/metric/triggers/anomalyAlertsForm.tsx

```diff
@@ -0,0 +1,156 @@
+};
+
+function SensitivityFormItem({
+  sensitivity,
+  onSensitivityChange,
+}: SensitivityFormItemProps) {
+  const handleSensitivityChange = ({value}) => {
+    onSensitivityChange(value);
+  };
+
+  return (
+    <StyledField
+      label={<div>{'Sensitivity'}</div>}
+      help={
+        <div>
+          {
+            'Lower sensitivity will alert you only when anomalies are larger, higher sensitivity will alert you and your team for even small deviations.'
+          }
+        </div>
+      }
+      required
+    >
+      <SelectContainer>
+        <SelectControl
+          name="sensitivity"
+          value={sensitivity}
+          options={[
+            {
+              value: AlertRuleSensitivity.LOW,
+              label: 'Low (alert less often)',
+            },
+            {
+              value: AlertRuleSensitivity.MEDIUM,
+              label: 'Medium',
+            },
+            {
+              value: AlertRuleSensitivity.HIGH,
+              label: 'High (alert more often)',
+            },
+          ]}
+          onChange={handleSensitivityChange}
+        />
+      </SelectContainer>
+    </StyledField>
+  );
+}
+
+function DirectionFormItem({
+  thresholdType,
+  onThresholdTypeChange,
+}: DirectionFormItemProps) {
+  const handleThresholdTypeChange = ({value}) => {
+    onThresholdTypeChange(value);
+  };
+
+  return (
+    <StyledField
+      label={<div>{'Direction'}</div>}
+      help={
+        <div>
+          {
+            'Indicate if you want to be alerted of anomalies above your set bounds, below, or both.'
+          }
+        </div>
+      }
+      required
+    >
+      <SelectContainer>
+        <SelectControl
+          name="sensitivity"
+          value={thresholdType}
+          options={[
+            {
+              value: AlertRuleThresholdType.ABOVE_AND_BELOW,
+              label: 'Above and below bounds',
+            },
+            {
+              value: AlertRuleThresholdType.ABOVE,
+              label: 'Above bounds only',
+            },
+            {
+              value: AlertRuleThresholdType.BELOW,
+              label: 'Below bounds only',
+            },
+          ]}
+          onChange={handleThresholdTypeChange}
+        />
+      </SelectContainer>
+    </StyledField>
+  );
+}
+
+class AnomalyDetectionForm extends Component<Props> {
+  render() {
+    const {sensitivity, onSensitivityChange, thresholdType, onThresholdTypeChange} =
+      this.props;
+
+    return (
+      <Fragment>
+        <SensitivityFormItem
+          sensitivity={sensitivity}
+          onSensitivityChange={onSensitivityChange}
+        />
+        <DirectionFormItem
+          thresholdType={thresholdType}
+          onThresholdTypeChange={onThresholdTypeChange}
+        />
+      </Fragment>
+    );
+  }
+}
+
+const StyledField = styled(FieldGroup)`
+  & > label > div:first-child > span {
+    display: flex;
+    flex-direction: row;
+  }
+`;
+
+const SelectContainer = styled('div')`
+  flex: 1;
```

Comment:
nit, there's a shared component for this in: `static/app/components/container/flex.tsx`. By reusing the shared components, we can reduce our javascript bundle size which really helps performance.